Add miss and tenn seals

quevon24 opened the pull request:
I added miss and tenn seals.

For the moment I added the mississippi judiciary seal while I wait for the response (I sent an email to the court asking for the seal) from the other seal I found:

![image](https://github.com/freelawproject/seal-rookery/assets/6474994/18c7a6c6-e1ff-468a-95e6-a88c3ff3f406)

flooie commented:
I think this isnt quite right.  The seal here is used by the workers comp boards but isnt the seal of the tenn. Supreme Court.   think you need to add the workers comp courts to the list and link that Agriculture seal to them - not add it to the Supreme Court here

quevon24 commented:
I already corrected the seals and, taking advantage of this, i added the correct one for Tennessee Supreme Court

quevon24 commented:
@flooie I think pre-commit.ci was configured for this repo but we never added the .pre-commit-config.yaml file, i added it and now precommit check works correctly

I also updated the checkout and setup-python actions in the workflow files
